app/src/main/res/values/styles.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <style name="AppTheme" parent="Theme.AppCompat.Light">
+    <style name="AppTheme" parent="Theme.AppCompat">
```

Comment:
Doesn't this break anything? 😓

Reply:
Errrr... This was accidental actually - every time I touch styling in this project I try this theme as a parent to try to understand why it was not used in the first place, but could not get to a conclusion yet... XD Do you have any info on that?

Reply:
I think that some activities have their own `Toolbar` in their layout, while some don't (the Firebase UI login one actually), so that's why the `.NoActionBar` one is used. Did I explain myself well? :D

Reply:
Sure, I got what you mean, makes sense. But I actually referred to the usage of ".Light" instead of the dark theme as a parent of the main AppTheme... Any reason for that that you know?

Reply:
Oh, sorry for the misunderstanding. I am not sure about that, probably just the color scheme used in the app better matches that?

Reply:
@eduardb told my through dm that it's probably due to FirebaseUI styling. Thanks, @eduardb!